src/serialization/JSONACSets.jl

```diff
@@ -179,8 +181,16 @@ The result is a JSON-able object (dictionary) from which a `JSONSchema.Schema`
 can be constructed, using the package JSONSchema.jl.
 """
 function acset_schema_json_schema(; kw...)
-  JSON.parsefile(joinpath(@__DIR__, "data", "acset.schema.json");
+  JSON3.parsefile(joinpath(@__DIR__, "data", "acset.schema.json");
                  dicttype=OrderedDict{String,Any}, kw...)
 end
 
+""" Dispatch for ACSet
+
+Dispatches write to accept ACSets
+"""
+JSON3.write(io::IO, acs::ACSet; kwargs...) = JSON3.write(io, generate_json_acset(acs); kwargs...)
```

Comment:
move this snippet up to near the other write method on line 93

Reply:
Done!